### PR TITLE
pnnx print flops memops count

### PR DESCRIPTION
**Pull request by SZUwishion**

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=5665) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=5665) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/ncnn?pullRequest=5665) it.</sub>

File: tools/pnnx/src/main.cpp [hunk 1]
```diff
@@ -382,6 +382,7 @@ int main(int argc, char** argv)
 
     //     pnnx_graph2.load("pnnx.param", "pnnx.bin");
     //     pnnx_graph2.save("pnnx2.param", "pnnx2.bin");
-
+    fprintf(stderr, "float ops = %dM\n", pnnx_graph.calculate_flops_M());
+    fprintf(stderr, "memory ops = %dM\n", pnnx_graph.calculate_memops_M());
     return 0;
```
Comment (nihui) [inline tools/pnnx/src/main.cpp:387]: 输出gflops/memops 应该在 `pnnx_graph.python(pnnxpypath, pnnxbinpath);` 后面

File: tools/pnnx/src/ir.cpp [hunk 2]
```diff
@@ -1441,6 +1442,48 @@ static std::string make_index_expression(const Operator* op)
     return index_expr;
 }
 
+int Graph::calculate_flops_M()
+{
+    long long flops = 0;
+    for(auto op:ops) {
+        if(op->type == "aten::matmul")
```
Comment (nihui) [inline tools/pnnx/src/ir.cpp:1449]: pnnx_graph python 之后的 op type 通常不会是 aten::XXX

**Comment by SZUwishion** [comment]
FLOPS参考：thop（[https://github.com/Lyken17/pytorch-OpCounter](url)）、ptflops（[https://github.com/sovrasov/flops-counter.pytorch](url)）
memops参考：torchstat（[https://github.com/sovrasov/flops-counter.pytorch](url)）

flops和memops相关运算：linear、pooling、conv、deconv、RNN系列、MultiheadAttention
其余操作计算为extra_flops和extra_memops

**Pull request closed without merging**